helix-core/src/main/java/org/apache/helix/task/TaskStateModel.java

```diff
@@ -283,6 +288,75 @@ public void reset() {
     }
   }
 
+  /**
+   * Loads className using classLoader
+   * @param classLoader
+   * @param className
+   * @return Class className loaded by classLoader
+   */
+  private Class loadClass(URLClassLoader classLoader, String className) {
+    try {
+      return classLoader.loadClass(className);
+    } catch (ClassNotFoundException e) {
+      LOG.error("Failed to load Task class " + className + " for new task in instance " + _manager
+          .getInstanceName() + " in cluster " + _manager.getClusterName() + ".");
+      throw new IllegalStateException("Class not found for task");
```

Comment:
Let's make sure error messages are consistent.

Reply:
Sorry, missed this one in the previous fix.